plugin.video.srgssr_ch_replay/resources/language/resource.language.en_gb/strings.po

```diff
@@ -33,19 +33,47 @@ msgid "Older episodes (page {0} of {1})"
 msgstr ""
```

Comment:
I'm thinking about just call ist "Next page ({0} of {1})" because it is not always older episodes e.g. when you go through search results or through movies.
@joachimBurket what do you think?

Reply:
I tested the add-on a little bit over lunch time and this kind of bothered me...

Reply:
I changed the English and German translation. What do you think and do you want to change the French one as well? Just push the changes to the branch.

Reply:
@joachimBurket  can you please provide us a sufficient translation?
msgid "Next page {0} of {1}"
msgstr "Épisodes plus anciens (page {0} of {1})"

msgid "Search broadcasts"
msgstr "Rechercher un épisode"

msgid "All"
msgstr "Sélectionner dans l'add-on"

msgid "Number of broadcasts per page"
msgstr "Nombre d'épisodes affichés par page"

Reply:
https://github.com/ManBehindMooN/kodi_plugin_video_srgssr_ch_replay/pull/15/commits/7a5364f15e9c0d88c095a724a23d49e41f6491bb > best guess 😃; I'm not sure if you want to translate broadcast to épisode... Let me know if you wish any changes.